src/applications/sah/sahg/helpers.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export const directDepositWarning = (
+  <div className="pension-dd-warning">
+    The Department of Treasury requires all federal benefit payments be made by
+    electronic funds transfer (EFT), also called direct deposit. If you don’t
+    have a bank account, you must get your payment through Direct Express Debit
+    MasterCard. To request a Direct Express Debit MasterCard you must apply at{' '}
+    <a
+      href="http://www.usdirectexpress.com"
+      target="_blank"
+      rel="noopener noreferrer"
```

Comment:
```suggestion
      rel="noopener noreferrer" aria-label="www.usdirectexpress.com (opens in new window)"
```

Reply:
Should this be changed in the generator as well?

Reply:
Fixed